config/default/kustomization.yaml

```diff
@@ -9,9 +9,9 @@ namespace: cluster-api-system
 namePrefix: cluster-api-
 
 bases:
-- crds/
-- rbac/
-- manager/
+- ../crds/
+- ../rbac/
+- ../manager/
```

Comment:
Wasn't there an issue with using `../` with kustomize v2?

Reply:
kustomize 2.0 is restricting `../` for `resources`, but not for `bases`,  https://github.com/kubernetes-sigs/kustomize/issues/794, i understand it could be confusing.

Reply:
Thanks for the clarification.